packages/office-ui-fabric-react/src/components/List/List.tsx

```diff
@@ -293,7 +293,7 @@ export class List extends BaseComponent<IListProps, IListState> {
     let divProps = getNativeProps(this.props, divProperties);
 
     // assign list if no role
-    role = role || 'list';
+    role = (typeof role === 'undefined') ? 'list' : role;
```

Comment:
`typeof null === 'object'`, is that what you're trying to test? Also, undefined shouldn't be in quotes. 

would you want this instead `role === undefined` which would yield false, passing null to role.
Then if role is not set, it would evaluate to true, passing 'list'.